sonar-project.properties

```diff
@@ -7,3 +7,5 @@ sonar.tests=test/
 
 sonar.sourceEncoding=UTF-8
 sonar.javascript.lcov.reportPaths=coverage/lcov.info
+
+sonar.exclusions=src/assets/javascript/**
```

Comment:
should this be excluded given that we wrote the js?

Reply:
i have made it more restrictive as all is something that is not owned by this team